examples/test_harness.sh

```
@@ -7,7 +7,7 @@ set +e
 
 [[ -f test/stdin.txt ]] && stdin_file="test/stdin.txt" || stdin_file="/dev/null"
 
-echo "[harness] running $1 $args < $stdin_file"
+[ $V -eq 1 ] && echo "[harness] running $1 $args < $stdin_file"
```

Shellcheck (0.11.0)

[info] Double quote to prevent globbing and word splitting.

(SC2086)